engine/editor/createcs350HUDLayer.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# ::----------------------------------------------------------------::
+# :: Copyright (c) 2010-2011 Zipline Games, Inc.
+# :: All Rights Reserved.
+# :: http://getmoai.com
+# ::----------------------------------------------------------------::
+
+${MOAI_BIN}/moai editor/createcs350HUDLayer.lua
+cp editor/generated/cs350HUDLayer.lua resources/layers
```

Comment:
This too. When we approach editing again let's make something more generic now that there is a strong pattern established.